ddlitlab2024/dataset/pytorch.py

```diff
@@ -40,15 +39,15 @@ class DDLITLab2024Dataset(Dataset):
     @dataclass
     class Result:
         joint_command: torch.Tensor
-        joint_command_history: torch.Tensor
-        joint_state: torch.Tensor
-        rotation: torch.Tensor
-        game_state: torch.Tensor
-        image_data: torch.Tensor
-        image_stamps: torch.Tensor
+        joint_command_history: Optional[torch.Tensor]
```

Comment:
While [https://peps.python.org/pep-0604/](equivalent) `type | None` is recommended for python 3.10+ https://peps.python.org/pep-0604/

Reply:
I'm a little confused why `joint_command` is not optional, but everything else is.
Is this `Result` used in two different ways, where maybe two diffferent data objects would make more sense?